pkg/sqlite/migrations/65_movie_group_rename.up.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE `movies` RENAME TO `groups`;
+
+DROP INDEX `index_movies_on_name`;
+CREATE INDEX `index_movies_on_name` ON `groups`(`name`);
```

Comment:
I assume this should be `index_groups_on_name`.

Reply:
Thanks!